### PR TITLE
Add FF3-1 Tweak Guidance Docs [VAULT-1036]

### PR DESCRIPTION
**Pull request by HridoyRoy**

### CODE REVIEW

File: website/content/docs/secrets/transform/ff3-tweak-details.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+However, this backfires when the number of tweaks used in FF3-1 setups
+is too small or is chosen on a restricted domain, or when it is user-generated and 
+access control on encrypting is too broad. In the former case, a codebook can still
+be easily created if the tweaks are known (since tweak values are not necessarily
+secret [3]). In the latter case, an attacker who gains access to encrypting data
+(ie, has chosen-plaintext strength) can manipulate the tweaks to essentially bypass
+that synthetic extension of the domain space. 
+
+FF3-1 uses 8 Feistel rounds, with a tweak schedule that works by summing
+(via abelian summation) the left and right sides of the tweak in an alternating
```
Comment (HridoyRoy) [inline website/content/docs/secrets/transform/ff3-tweak-details.mdx:36]: This paragraph may be unnecessary.

File: website/content/docs/secrets/transform/ff3-tweak-details.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+stored with the same tweaks. For example, if the ciphertexts corresponding to the last four digits
+of phone numbers are stored, then there may be plaintext collisions, and these
+collision values should not be encrypted with the same tweak [3, Appendix C].
+
+## FF3-1 Best Practices:
+
+From the problems that may arise from the previous section, we have a list
+of recommendations for how to use tweaks in the FF3-1 paradigm.
+
+* Encryption should be as restricted as possible via ACLs.
```
Comment (HridoyRoy) [inline website/content/docs/secrets/transform/ff3-tweak-details.mdx:64]: This should be front and center.

File: website/content/docs/secrets/transform/ff3-tweak-details.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+As FF3-1 splits tweaks in half and uses half a tweak in each round function,
+randomly chosen tweaks should have high entropy within each half of the tweak.
+For example, creating many encryptions with tweaks with the same right or left half 
+will lead to more biased encryptions (where the distinction between the corresponding
+encryptions rely solely on the randomness of the abelian summation operator).
+In practice, using randomly chosen tweaks will be enough -- however, it is important
+to avoid using an algorithm to generate tweaks that yields similarity between two
+halves of tweaks [4]. 
+
+Finally, encrypting the same plaintext with the same key and tweak is deterministic,
```
Comment (HridoyRoy) [inline website/content/docs/secrets/transform/ff3-tweak-details.mdx:52]: This should be front and center too.

File: website/content/docs/secrets/transform/ff3-tweak-details.mdx [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# FF3-1 Tweak Usage Documentation
+
```
Comment (HridoyRoy) [inline website/content/docs/secrets/transform/ff3-tweak-details.mdx:2]: Mention in the doc that this is only for supplied and internal sourcing.

File: website/content/docs/secrets/transform/ff3-tweak-details.mdx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+## FF3-1 Tweak Usage
+
+FF3-1 is a tweakable encryption scheme that was created to solve the problem of
+encrypting on small domains. Tweaks were introduced to synthetically increase the
+domain space. With the introduction of tweaks, an attacker must now have the
+correct encryption of the plaintext and the correct tweak in order to backtrack 
+via a codebook from an encryption to a plaintext. 
+
+However, this backfires when the number of tweaks used in FF3-1 setups
```
Comment (sgmiller) [inline website/content/docs/secrets/transform/ff3-tweak-details.mdx:47]: s/backfires/can be risky/  In general go for more neutral phrasing.  Backfires might cause someone to assume immediate failure

**Review by sgmiller — COMMENTED** [review]
Minor comment but I like the restructuring.

File: website/content/docs/secrets/transform/ff3-tweak-details.mdx (hunk 5 above)
Comment (HridoyRoy) [reply website/content/docs/secrets/transform/ff3-tweak-details.mdx]: Definitely, good catch! Fixed!

**Review by sgmiller — APPROVED** [review]

**Pull request merged**